zebra-consensus/src/primitives/groth16.rs

```diff
@@ -68,6 +68,22 @@ pub type BatchVerifyingKey = VerifyingKey<Bls12>;
 /// This is the key used to verify individual items.
 pub type ItemVerifyingKey = PreparedVerifyingKey<Bls12>;
 
+lazy_static::lazy_static! {
+    /// The Sapling Groth16 verifying key for spends.
+    pub static ref SAPLING_SPEND_VERIFYING_KEY: std::sync::OnceLock<crate::groth16::VerifyingKey<Bls12>> = {
+        let cell = std::sync::OnceLock::new();
+        let _ = cell.set(GROTH16_PARAMETERS.sapling.spend.verifying_key().inner().clone());
+        cell
+    };
+
+     /// The Sapling Groth16 verifying key for outputs.
+    pub static ref SAPLING_OUTPUT_VERIFYING_KEY: std::sync::OnceLock<crate::groth16::VerifyingKey<Bls12>> = {
+        let cell = std::sync::OnceLock::new();
+        let _ = cell.set(GROTH16_PARAMETERS.sapling.output.verifying_key().inner().clone());
+        cell
+    };
+}
```

Comment:
I had problems with the static lifetimes so i had to add this but there is maybe an easier way.

Reply:
This looks like good use of `lazy_static`, I can't think of an easier way, but since these values are being cloned anyways and are only used once, using references doesn't seem to have any advantage over providing the batch verifier with an owned copy of the verifying key, so that's an option too.